docs/source/usage/cli.rst

```diff
@@ -0,0 +1,156 @@
+Commandline Interface
+=====================
+Metasyn provides a command line interface (CLI) for accessing core functionality without needing to write any Python code.
+
+The CLI currently has two subcommands. The ``synthesize`` subcommand, which allows you to **generate synthetic data** from any ``GMF file``, and the ``schema`` subcommand, which allows you to **create validation schemas** for a :obj:`MetaFrame<metasyn.metaframe.MetaFrame>`.
```

Comment:
The validation schema should be for the GMF file, not the MetaFrame I think.

Reply:
You're completely right